dependencyfile.py

```diff
@@ -16,7 +16,8 @@
 from os import remove
 from shutil import copyfile
 from defusedxml.ElementTree import parse, XMLParser
-import xml.etree.ElementTree as ET  # Not using this module in an unsafe way -- skipcq: BAN-B314
+# import xml.etree.ElementTree as ET  # Not using this module in an unsafe way -- skipcq: BAN-B405
```

Comment:
@jamie-meyer do you need to include this if it is no longer being used?

Reply:
Oh that's totally my bad. I accidentally commented it out.